### PR TITLE
Update heroku.rst

### PR DESCRIPTION
**Pull request by adyassine**

### CODE REVIEW

File: deployment/heroku.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ Preparing your Application
 Deploying a Symfony application to Heroku doesn't require any change in its
 code, but it requires some minor tweaks to its configuration.
 
-By default, the Symfony app will log into your application's ``app/log/``
+By default, the Symfony app will log into your application's ``bin/log/``
```
Comment (xabbuh) [inline deployment/heroku.rst]: Didn't you mean `var/log/`?

**Review by xabbuh — CHANGES_REQUESTED** [review]

**Comment by adyassine** [comment]
Yes I'm sorry

File: deployment/heroku.rst (hunk 1 above)
Comment (adyassine) [inline deployment/heroku.rst]: Yes, I'm sorry

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by xabbuh** [comment]
Good catch, thanks @adyassine.

**Pull request closed without merging**